initfiles/componentfiles/configxml/esp.xsd.in

```diff
@@ -408,11 +408,11 @@
 	                </xs:appinfo>
 	              </xs:annotation>
 	            </xs:attribute>
-	            <xs:attribute name="resourceURL" type="xs:string" use="optional" default="/favicon.ico,/esp/files/img/favicon.ico,/esp/files/eclwatch/img/Loginlogo.png,/esp/files/dojo/*,/esp/files/eclwatch/nls/*">
+	            <xs:attribute name="unrestrictedResources" type="xs:string" use="optional" default="/favicon.ico,/esp/files/img/favicon.ico,/esp/files/eclwatch/img/Loginlogo.png,/esp/files/dojo/*,/esp/files/eclwatch/nls/*">
```

Comment:
Is there anything under /esp/files that we want to protect?  They are mostly if not all checked into opensource anyway, right?  Perhaps the default should be /esp/files/* instead of these individual sub directories.  Also perhaps /esp/xslt?  I assume these will prevent calls to DALI.